src/main/java/com/epam/task/purchases/Euro.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.task.purchases;
+
+public class Euro implements Comparable<Euro>{
+    private final int value;
+
+    public Euro(int value) {
+        this.value = value;
+    }
+
+    public Euro(int euros, int cents){
+        this.value = euros*100 + cents;
```

Comment:
use the primary ctor. 
```suggestion
        this(euros*100 + cents);
```
it's better for validation purpose for example.